File: hello-word/src/types/Municipality.ts
import { City } from "./City";

export class Municipality {

    id: string
    private name: string
    private description: string
    city!: City;

    constructor(id: string, name: string, description: string) {
        this.id = id;
        this.name = name
        this.description = description
    }

}

File: cityapi/src/main/kotlin/com/example/demo/entities/Municipality.kt
package com.example.demo.entities

import com.fasterxml.jackson.annotation.JsonIgnore
import org.hibernate.annotations.GenericGenerator
import javax.persistence.*

@Entity
@Table(name = "municipalities")
data class Municipality(
    @Id
    @GenericGenerator(name = "uuid2", strategy = "uuid2")
    @GeneratedValue(strategy = GenerationType.IDENTITY, generator = "uuid2")
    var id: String?,
    val name: String,
    var description: String?,
    @ManyToOne
    @JoinColumn(name = "city_id")
    @JsonIgnore
    var city: City?
)
